compiler/optimizer/StructuralAnalysis.cpp

```diff
@@ -51,7 +51,7 @@ void TR_RegionAnalysis::simpleIterator (TR_Stack<int32_t>& workStack,
                                         bool doThisCheck)
    {
    SparseBitVector::Cursor cursor(vector);
-   for (cursor.SetToLastOne(); cursor.Valid(); cursor.SetToPreviousOne())
+   for (cursor.SetToFirstOne(); cursor.Valid(); cursor.SetToNextOne())
```

Comment:
Do sparse and dense bitvectors order their bits differently? Or is this a separate optimization?